src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -130,7 +130,7 @@ def acr_update_get(client,
     )
 
 def acr_update_custom(instance,
-                      admin_enabled=None,
+                      admin_enabled,
```

Comment:
Do we want to disallow
"acr update --admin-enabled"
but allow
"acr crate --admin-enabled"

Would that be confusing?